### PR TITLE
New review button

### PR DESCRIPTION
**Pull request by kcirerick**
<ul>
<li>Adds button to add a new review on results page.</li>
<li>Displays new review form on modal while hiding everything else.</li>
<li>Enables user to go from new review form to results page, but not to reviews page. Users can exit from any of the three pages and continue their search normally.</li>
</ul>

### CODE REVIEW

File: src/main/webapp/index-style.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import url(modal-overlay.css);
+@import url(newReview.css);
```
Comment (dterr) [inline src/main/webapp/index-style.css:2]: Nice use of ```import``` to keep our files shorter.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -48,10 +48,58 @@
...
         </div>
         <div class="modal-body" id="results-body">
-          <div id="results-list-container"></div>
+          <div class="list" id="results-list-container"></div>
         </div>
         <div class="modal-body" id="reviews-body">
-          <div id="reviews-list-container"></div>
+          <div class="list" id="reviews-list-container"></div>
+        </div>
+        <div class="modal-body" id='rev-form-body'>
```
Comment (dterr) [inline src/main/webapp/index.html:56]: Where did the rest of the modal body go? The elements that are populated for the search results and the review results.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -48,10 +48,58 @@
           <button data-modal-close-button class="exit-button">&times;</button>
         </div>
         <div class="modal-body" id="results-body">
-          <div id="results-list-container"></div>
+          <div class="list" id="results-list-container"></div>
         </div>
         <div class="modal-body" id="reviews-body">
-          <div id="reviews-list-container"></div>
+          <div class="list" id="reviews-list-container"></div>
```
Comment (dterr) [inline src/main/webapp/index.html:54]: Why make this of the list class? I'm unfamiliar with it.

Comment (kcirerick) [reply src/main/webapp/index.html:54]: "list" is a made up name, I just needed to access two specific elements in js so I assigned them the same class. It made the js smoother.

File: src/main/webapp/index.html (hunk 2 above)
Comment (kcirerick) [reply src/main/webapp/index.html:56]: I'm not really sure what you mean by this? It populates well right now?

Comment (dterr) [reply src/main/webapp/index.html:56]: Nevermind. Upon further inspection of the file I see my comment makes no sense. From the code snippet it showed me I thought you deleted code.

**Pull request merged**